go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/openshift/source-to-image
 go 1.13
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20170808145631-06b9cb351610
-	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
-	github.com/moby/buildkit => github.com/dmcgowan/buildkit v0.0.0-20170731200553-da2b9dc7dab9
+	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.2.0
+	github.com/containerd/containerd => github.com/containerd/containerd v1.3.0
+	github.com/docker/docker => github.com/docker/docker v0.0.0-20180522102801-da99009bbb11
```

Comment:
Docker has been downgraded here - why?

Reply:
To use the same Docker version `containers/image` is using:

https://github.com/containers/image/blob/v5.0.0/go.mod#L13